include/mp/util.h

```diff
@@ -238,6 +244,24 @@ inline char* CharCast(unsigned char* c) { return (char*)c; }
 inline const char* CharCast(const char* c) { return c; }
 inline const char* CharCast(const unsigned char* c) { return (const char*)c; }
 
+#if defined(__GNUG__)          // GCC & Clang ─ use <cxxabi.h> to demangle
```

Comment:
I don't love this without an escape-hatch for more esoteric compilers. Something like `#if defined(__GNUG__) && !defined(MP_NO_DEMANGLE)` would make it more bulletproof.

That can always be added if it ever becomes a real problem though.